napari/components/viewer_model.py

```diff
@@ -1001,12 +1002,19 @@ def _get_image_class() -> layers.Image:
     return layers.Image
 
 
-def _start_monitor(layers: LayerList) -> None:
+def _create_remote_commands(layers: LayerList) -> None:
```

Comment:
I'm ok with this, it is private and we can revisit in future

Reply:
I could imagine some better pattern for experimental features in general. I think our experimental stuff is working great, but there has to be some "feature toggle" place where non-experimental activates experimental, and we seem to do it a few different ways, and none seem super clean.

Reply:
Maybe use utils/config.py more. Our general approach is guard with an env var, and don't even import the experimental code unless it says too. Maybe guarding with config.py more would help somehow, but now sure how.

Reply:
> I think our experimental stuff is working great, but there has to be some "feature toggle" place where non-experimental activates experimental, and we seem to do it a few different ways, and none seem super clean.

Yes, this will be critical.